docs/issue-templates/README.MD

```diff
@@ -0,0 +1,21 @@
+# Standard issue responses
+
+Standard issue responses to be used by the ZAP Core Team.
+
+For ease of use they can be manually added to https://github.com/settings/replies 
+
+### Question As Issue
+Please do not ask questions as issues.
+
+The ZAP User Group is the place to ask them: https://groups.google.com/g/zaproxy-users
```

Comment:
Could we add something like:
```
If you’re asking about a third party components such as the Jenkins plugin or VSTS plugin please look for their preferred support mechanism as these were not created by and are not supported by the core team.
```

which can be left or trimmed from the end easily.

Reply:
Do we get many questions about 3rd party components? I cant remember many.
I'd prefer this was a separate answer tbh at I would write something different. I'll see if I can come up with something...

Reply:
How about that?